fontforge/python.c

```diff
@@ -1735,50 +1735,100 @@ return( Py_BuildValue("i",ret));
 }
 
 static PyObject *PyFF_askChoices(PyObject *UNUSED(self), PyObject *args) {
-    char *title=NULL,*quest=NULL, **answers;
-    int def=0, cnt;
-    PyObject *answero;
-    int i, ret;
+    char *title=NULL, *quest=NULL;
+    char **answers; // receives answers written into `answero`
+    PyObject* defo = NULL; // default answer index, or tuple of len cnt
+    int def = 0; // default index, only used if not multianswer
+    int cnt; // number of answers
+    PyObject *multipleo; // expected to be boolean
+    bool multiple = false;
+    PyObject *answero; // expected to be tuple
+    int ret;
 
     if ( no_windowing_ui ) {
-	PyErr_Format(PyExc_EnvironmentError, "No user interface");
-return( NULL );
+        PyErr_Format(PyExc_EnvironmentError, "No user interface");
+        return( NULL );
     }
 
-    if ( !PyArg_ParseTuple(args,"esesO|i","UTF-8", &title, "UTF-8", &quest, &answero, &def) )
-return( NULL );
+    if ( !PyArg_ParseTuple(args,"ssO|OO", &title, &quest, &answero, &defo, &multipleo) )
+        return( NULL );
+
+    multiple = multipleo == Py_True;
```

Comment:
If we were python-3-only we could use `p` with a C int  here instead of `O` for the booleans. They didn't backport `p` to Python 2, unfortunately. 

However, what we care about is not whether these various booleans are identical to `Py_True`, we care if they are *true* in the Python sense. So replace `multipleo == Py_True` with `PyObject_IsTrue(multipleo)`.

Reply:
> If we were python-3-only we could use p with a C int here instead of O for the booleans. They didn't backport p to Python 2, unfortunately.

I think we're almost at the point where we should just drop Python 2 support entirely (not that the build system currently supports it anyway), which would allow us that.

Reply:
I do not personally object -- at all -- to dropping python 2 support soon, or even now. However, I have the sense that in the near term we'll still face significant pushback on that move.

Reply:
Yes. Let's wait a bit more before we burn that bridge. I find it encouraging, though, that there has been no complaint about crossing it.